modules/custom/core/govcms_content_types/config/install/workflows.workflow.editorial.yml

```diff
@@ -0,0 +1,76 @@
+langcode: en
```

Comment:
Config for Workflow is moved to `govcms_content_types` so that it could be imported *before* any GovCMS default content type module is installed. Currently it is a config of the profile, which is imported *after* all other dependencies.